dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/Parent.java

```diff
@@ -115,28 +121,40 @@ protected void forkRepositoriesFound(Map<String, String> parentToPath,
              * repositories that were automatically fixed by GitHub. Instead, we save the names of the parent repos
              * in the map above, find the list of repositories under the authorized user, and iterate through that list.
              */
-            GHRepository parent = c.getOwner();
+            parent = c.getOwner();
+            parentRepoName = parent.getFullName();
             if (parent.isFork()) {
                 log.warn("Skipping {} because it's a fork already. Sending a PR to a fork is unsupported at the moment.",
-                        parent.getFullName());
+                        parentRepoName);
             } else {
-                log.info("Forking {}...", parent.getFullName());
```

Comment:
Did this move?

Reply:
[Just 2 lines below](https://github.com/salesforce/dockerfile-image-update/pull/25/files#diff-da5f0732ed5f695522bdefddf18a2555R133) :)

Reply:
Uh oops... /Runs away

Thanks! 😆